test/azmi-main-tests/blob/GetBlobs-Tests.cs

```diff
@@ -0,0 +1,143 @@
+﻿using azmi_main;
+using Azure;
+using Azure.Storage.Blobs.Models;
+using NSubstitute;
+using NSubstitute.ExceptionExtensions;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace azmi_tests
+{
+    public class GetBlobs_Tests
+    {
+        [Fact]
+        public void DummyTest()
+        {
+            Assert.True(true);
+        }
+
+        public class SharedArguments_TestsGroup
+        {
+            [Fact]
+            public void IdentityArgument()
+            {
+                var subCommand = new GetBlobs();
+                var arguments = subCommand.Definition().arguments;
+                Assert.NotEmpty(arguments.Where(a => a.name == SharedAzmiArguments.identity.name));
+            }
+
+            [Fact]
+            public void VerboseArgument()
+            {
+                var subCommand = new GetBlobs();
+                var arguments = subCommand.Definition().arguments;
+                Assert.NotEmpty(arguments.Where(a => a.name == SharedAzmiArguments.verbose.name));
+            }
+        }
+
+        public class RealArguments_TestsGroup
+        {
+            // test if it has real arguments we defined, prevents accidental argument removal
+            [Theory]
+            [InlineData("container")]
+            [InlineData("directory")]
+            [InlineData("if-newer")]
+            [InlineData("prefix")]
+            [InlineData("delete-after-copy")]
+            [InlineData("exclude")]
+            // TODO: Uncomment this later [InlineData("absolute-paths")]
```

Comment:
Valid or not?

Reply:
Yes, still valid. I need to merge this change with yours where you added it. Currently, I do not have that param in my branch.